internal/mode/static/manager.go

```diff
@@ -188,15 +188,12 @@ func StartManager(cfg config.Config) error {
 		)
 	}
 
-	statusUpdater := status.NewUpdater(status.UpdaterConfig{
-		GatewayCtlrName:          cfg.GatewayCtlrName,
-		GatewayClassName:         cfg.GatewayClassName,
-		Client:                   mgr.GetClient(),
-		Logger:                   cfg.Logger.WithName("statusUpdater"),
-		Clock:                    status.NewRealClock(),
-		UpdateGatewayClassStatus: cfg.UpdateGatewayClassStatus,
```

Comment:
What happened to the functionality around this field? Is it still used anywhere?

Reply:
good catch. I unintentionally removed that feature while retaining the config.

We added this feature when added the provisioner mode, because in that case, the provisioner is responsible for updating statuses of GatewayClass.

I will restore the implementation of this feature.